mscore/file.cpp

```diff
@@ -2791,7 +2791,7 @@ bool MuseScore::saveSvg(Score* score, const QString& name, const NotesColors& no
             QString fileName(name);
             if (fileName.endsWith(".svg"))
                   fileName = fileName.left(fileName.size() - 4);
-            fileName += QString(" - %1.svg").arg(pageNumber+1, padding, 10, QLatin1Char('0'));
+            fileName += QString("-%1.svg").arg(pageNumber+1, padding, 10, QLatin1Char('0'));
```

Comment:
this looks like an old leftover, those spaces arounf the - had been removed for PNG a long time ago IIRC